src/elements/common/sub-header/SortButton.js

```diff
@@ -5,19 +5,22 @@
  */
 
 import React from 'react';
-import { FormattedMessage } from 'react-intl';
+import { injectIntl } from 'react-intl';
 import Button from '../../../components/button';
 import IconSort from '../../../icons/general/IconSort';
 import messages from '../messages';
 import Tooltip from '../Tooltip';
 import './SortButton.scss';
 
-const SortButton = (props: ?Object) => (
-    <Tooltip text={<FormattedMessage {...messages.sort} />}>
-        <Button className="be-btn-sort" type="button" {...props}>
-            <IconSort />
-        </Button>
-    </Tooltip>
-);
+const SortButton = ({ intl, ...props }: ?Object) => {
```

Comment:
small: I would do rest instead of props
```suggestion
const SortButton = ({ intl, ...rest }: ?Object) => {
```

Reply:
good point!